code/src/components/Game.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import {
+	ConsoleContent,
+	StartButton,
+	StartButtonImg,
+	GameTextHeader,
+	Span,
+	LabelName,
+	UsernameInput,
+	UsernameInputContainer,
+	Modal,
+} from './Styling'
```

Comment:
Really good that you have put styling to its own component, so the code keeps clean and it's well organized this way.